test/detect-buffer-noassert.js

```diff
@@ -0,0 +1,40 @@
+var RuleTester = require("eslint").RuleTester;
+
+var rule = require("../rules/detect-buffer-noassert");
+
+
+var valid = "a.readUInt8(0);",
+  valid2 = "a.readUInt8(0, false);",
+  invalid = "a.readUInt8(0, true);",
+  eslintTester = new RuleTester();
+
+
+eslintTester.run("detect-buffer-noassert", rule, {
+  valid: [{ code: valid }],
```

Comment:
Sure! I've removed the ones with the strings we're not reusing.

https://github.com/nodesecurity/eslint-plugin-security/pull/3/commits/bc4de92e0025f7fb78e1dcbeae15aad64b867312